build.gradle

```diff
@@ -58,12 +58,12 @@ subprojects {
     version = scmVersion.version
 
     dependencies {
-        api group: 'org.springframework', name: 'spring-web', version: '5.3.6'
+        api group: 'org.springframework', name: 'spring-web', version: '6.0.2'
         api group: 'io.github.resilience4j', name: 'resilience4j-circuitbreaker', version: resilience4jVersion
```

Comment:
Please update all the dependencies.

Reply:
added more updates, please check
also added you guys as collaborators to make this work faster ;)

Reply:
Please check if everything is ok. As for my needs, the current build works ok